src/database.ts

```diff
@@ -235,9 +235,31 @@ async function normalizedToRunnables(
     );
   }
   if (normalized.tables.length > 1) {
```

Comment:
here, we check if there are > 1 tables in the normalized statement. if so, we will:
- renormlize each statement
- check for a SELECT FROM statement and get the table name
- filter the `norm.tables` array and remove this table from the array, stored as `filteredTables`
- if the length of `filteredTables` is > 1, that means there's more than one table trying to be mutated, which throws

the one callout: as noted [here](https://linear.app/tableland/issue/JSTC-34#comment-b6c4917e), we **don't** want to allow a query like this, shown using pg:
```
with ins as (
    insert into table1 (userid, email)
    values(12,'example@gmail.com') 
)
insert into table2 (userid, name, address)
values(12, 'Joe', '123 test Dr')
```
i could be wrong, but i don't think there's a way to do this in SQLite without passing multiple queries in a single runnable—i.e., `INSERT INTO table1 ...; INSERT INTO table2 ...;` will throw. so, i _think_ the `filteredTables` logic to remove a table should work. in other words, if the pg query was valid in SQLite, this logic would be flawed.

--

also, there could be more efficient ways to do this—open to suggestions.

Reply:
I spent some time trying to figure out how to do something like the above pg query with sqlite and I can't come up with anything.